CONTRIBUTING.md

```diff
@@ -198,3 +198,9 @@ Then run the following to log into the GitHub container registry, entering your
 ```
 docker login ghcr.io
 ```
+
+### How to: Run the BaaS server locally
+
+It can be useful to run the BaaS server directly, rather than in the docker container, for example to use the latest version from a branch or for debugging.
+
+You can do this by running: `AWS_ACCESS_KEY_ID="???" AWS_SECRET_ACCESS_KEY="???" ./evergreen/install_baas.sh -w ../../../baas-work-dir -b master` from the `vendor/realm_core` directory. Logs can be accessed in `../../../baas-work-dir/baas_server.log`.
```

Comment:
It can, although it should probably be `./vendor/realm-core/evergreen/install_baas.sh -w ../baas-work-dir -b master` as I think it makes sense for the `baas-work-dir` to be outside of the git repo

Reply:
I'm fine with that.  Was just a bit unclear where I was with the `../../../`